salt/search/config/srv-search-config.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+use Monolog\Logger;
+
+return [
+    'debug' => {% if pillar.search.debug %}true{% else %}false{% endif %},
+    'validate' => {% if pillar.search.validate %}true{% else %}false{% endif %},
+    'ttl' => {{ pillar.search.ttl }},
+    'elastic_servers' => ['{{ pillar.search.elasticsearch.servers }}'],
+    'elastic_logging' => {% if pillar.search.elasticsearch.logging %}true{% else %}false{% endif %},
+    'elastic_force_sync' => {% if pillar.search.elasticsearch.force_sync %}true{% else %}false{% endif %},
+    'gearman_servers' => ['{{ pillar.search.gearman.servers }}'],
```

Comment:
Pillar name is confusing as it's a single value not a list.

Reply:
Yes, we never use a cluster configuration for Elasticsearch or Gearman so this should all be singular for simplicity.